### PR TITLE
Create .sync.yml

### LINKED ISSUE #40: Add Travis Beaker tests back
From reading the modulesync docs, it should be a matter of adding this a .sync.yml:

```yaml
---
.travis.yml:
  includes:
  - rvm: '2.1.6'
    sudo: required
    services: docker
    env: PUPPET_VERSION="3.8.4" BEAKER_set="centos-6-docker-x64"
    script: bundle exec rake acceptance
    bundler_args: --without development
  - rvm: '2.1.6'
    sudo: required
    services: docker
    env: PUPPET_VERSION="3.8.4" BEAKER_set="ubuntu-server-1204-docker-x64"
    script: bundle exec rake acceptance
    bundler_args: --without development
```

Then running modulesync again...

### PR DESCRIPTION
**Pull request by petems**
* Customise .travis.yml file to re-add Beaker tests

Closes #40

### CODE REVIEW

File: .sync.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - rvm: '2.1.6'
+    sudo: required
+    services: docker
+    env: PUPPET_VERSION="3.8.4" BEAKER_set="centos-6-docker-x64"
+    script: bundle exec rake acceptance
+    bundler_args: --without development
+  - rvm: '2.1.6'
+    sudo: required
+    services: docker
+    env: PUPPET_VERSION="3.8.4" BEAKER_set="ubuntu-server-1204-docker-x64"
```
Comment (jyaworski) [inline .sync.yml]: Can you use the ~> syntax on 3.8.0 instead? Also, should we be testing on 12.04? Would 14.04 be a better target?

Comment (petems) [inline .sync.yml]: I mean 12.04 is in LTS until 2017. We could probably do both? Maybe CentOS 6 and 7, Ubuntu 12.04 and 14.04, covers most bases.

**Comment by jyaworski** [comment]
@petems can you look at the tests failing? A rebase on master may solve it.

**Comment by bastelfreak** [comment]
ping @petems  :)

**Comment by petems** [comment]
Leaving this for now

**Pull request closed without merging**